xformers/ops.py

```diff
@@ -239,22 +239,6 @@ def supports(cls, d: "AttentionOpDispatch") -> bool:
             matmul_alignment_mn = max(matmul_alignment_mn, 64 // bits_per_scalar)
         if d.k % matmul_alignment_mn != 0:
             return False
-
```

Comment:
This is awesome!

Do you think it would make sense to add some more test cases which would fail before but passes now? Or are the test cases covering everything?

Reply:
I believe it's already tested:
```
@pytest.mark.parametrize("k_len", [5, 6, 32, 128])
@pytest.mark.parametrize("batch_size", [1, 4])
@pytest.mark.parametrize("kv_len", [3, 15, 32, 34, 64, 256])
@pytest.mark.parametrize("q_len", [2, 3, 5, 32, 34, 256])
@pytest.mark.parametrize("device", _devices)
@pytest.mark.parametrize("dtype", [torch.float, torch.half, torch.bfloat16])
```